fastmri/data/mri_data.py

```diff
@@ -358,3 +360,153 @@ def __getitem__(self, i: int):
             sample = self.transform(kspace, mask, target, attrs, fname.name, dataslice)
 
         return sample
+
+
+class AnnotatedSliceDataset(SliceDataset):
+    def __init__(
+            self,
+            root: Union[str, Path, os.PathLike],
+            challenge: str,
+            mri_type: str,
+            multiple_annotation_policy: str,
+            transform: Optional[Callable] = None,
+            use_dataset_cache: bool = False,
+            sample_rate: Optional[float] = None,
+            volume_sample_rate: Optional[float] = None,
+            dataset_cache_file: Union[str, Path, os.PathLike] = "dataset_cache.pkl",
+            num_cols: Optional[Tuple[int]] = None,
+            annotation_version: Optional[str] = None,
+    ):
+        """
+        Args:
+            root: Path to the dataset.
+            challenge: "singlecoil" or "multicoil" depending on which challenge
+                to use.
+            mri_type: 'knee' or 'brain' depending on which dataset to use.
```

Comment:
Could we rename this variable to `subsplit` or `anatomy`? `mri_type` is too vague. Use a refactoring tool to prevent bugs.

Reply:
Done.